### ISSUE TITLE
Kernel panic at boot

### ISSUE THREAD

**Issue by Jonas-July** [OP-TEE/optee_os#5625]
#### General context
I'm running OpTEE on the Xilinx Versal VMK180 FPGA. The image was created using the [guide in the docs](https://optee.readthedocs.io/en/latest/building/devices/versal.html). I then [put the files on an SD card](https://optee.readthedocs.io/en/latest/building/devices/versal.html#sd-card-creation-and-boot) and put the SD card into the Versal slot on the board.
This is more of an issue for [foundriesio/optee_os](https://github.com/foundriesio/optee_os), but I can't open an issue there, so I hope that one of the developers answers here.

#### What is the issue?
When starting up the FPGA, the image is booted, but shortly afterwards experiences a kernel panic with the following relevant error message:
```
I/TC: Hardware Root of Trust:	Asymmetric NOT Enabled, Symmetric NOT Enabled
D/TC:0 0 call_initcalls:40 level 3 sha3_init()
E[T830.0 5ersal_mbox_notify:318 Can: Error:the remote response for Comm0 0 versal_21
[184i.0:5]PLM Error Status: 0x252101F4
b_status = 0xf4
E/TC:0 0 Panic at core/drivers/crypto/versal/hash.c:241 <sha3_init>
E/TC:0 0 TEE load address @ 0x60000000
E/TC:0 0 Call stack:
E/TC:0 0  0x60008818
E/TC:0 0  0x6001a04c
E/TC:0 0  0x60018d5c
E/TC:0 0  0x6001b9ac
E/TC:0 0  0x60008658
```
Noteworthy are the Error Status `0x252101F4` and the line `Panic at core/drivers/crypto/versal/hash.c:241 <sha3_init>`
A possibly related issue is that the system immediately boots into Linux, without stopping at U-Boot.
See also [the complete log](https://github.com/OP-TEE/optee_os/files/9938254/optee-os-error.log)

#### How to reproduce the issue?
Basically follow the guide above with some modifications from [this issue](https://github.com/OP-TEE/optee_test/issues/622), but one needs to have a VMK180 FPGA available.

Thanks for your time and any help you can provide

**Comment by Jonas-July** [comment]
My board is encryption-disabled, I don't know if that means that SHA3 is unavailable.
